src/identifiers.md

```diff
@@ -21,5 +23,7 @@ Or
 * The identifier is more than one character. `_` alone is not an identifier.
 * The remaining characters are alphanumeric or `_`.
 
+A raw identifier is like a normal identifier, but prefixed by `r#`, which is not included as part of the identifier. Unlike a normal identifier, a raw identifier may be any strict or reserved keyword.
```

Comment:
Nit: Comma splice. Trade the last `,` for a `.` and replace `which` with ```The `r#` ```

Reply:
Hmm I think you misread perhaps? It wouldn't make sense like that grammatically.

Reply:
```A raw identifier is like a normal identifier, but prefixed by `r#`. The `r#` is not included as part of the identifier.```

Reply:
I'm not sure that improves clarity, but if you really want, I'll do it. 😉

Reply:
Actually, looking at it closer, another option is to change ", but" to "except". So ```A raw identifier is like a normal identifier except prefixed by `r#`, which is not included as part of the identifier.``` The main thing is to get rid of one of the commas as it's currently grammatically murky with both of them.